addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/ParameterTamperScanRule.java

```diff
@@ -240,4 +237,39 @@ public int getCweId() {
     public int getWascId() {
         return 20;
     }
+
+    private AlertBuilder createAlert(int confidence, String param, String attack, String sb, HttpMessage msg) {
+        return newAlert()
+                .setConfidence(confidence)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(sb)
+                .setMessage(msg);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        List<Alert> alerts = new ArrayList<>();
```

Comment:
Use `List.of`.

Reply:
Return directly instead of intermediately storing.